### DISCUSSION TITLE
BurgerPanel 1.0.5

### DISCUSSION THREAD

**Discussion by TheBlueBurger** [TheBlueBurger/BurgerPanel#9] [Announcements]
Get port from env, set up admin user on first launch, fix bug in logger causing debugs to display as info, remove "serverPathIsRelative" and fix bug with server error handler

<hr /><em>This discussion was created from the release <a href='https://github.com/TheBlueBurger/BurgerPanel/releases/tag/1.0.5'>BurgerPanel 1.0.5</a>.</em>

**Comment by TheRedXD** [comment]
Best panel
